### PR TITLE
Updating Regulation Translation Index Page

### PR DESCRIPTION
**Pull request by hanwu85**
Updating translation status for Aug. 1st 2020 version for the following languages.

[Chinese ](https://github.com/thewca/wca-regulations-translations/pull/90)
[Finnish](https://github.com/thewca/wca-regulations-translations/pull/93)
[Portuguese Brazil](https://github.com/thewca/wca-regulations-translations/pull/89)
[Chinese, Traditional](https://github.com/thewca/wca-regulations-translations/pull/96)
[Russian](https://github.com/thewca/wca-regulations-translations/pull/91)

### CODE REVIEW

File: WcaOnRails/app/views/regulations/translations/index.html.erb [hunk 1]
```diff
@@ -10,11 +10,31 @@
...
+    },
+  ] %>
+
+  <h3><%= t("regulations_translations.current") %></h3>
+  <div class="regulations-translations-table-container">
+    <%= render 'regulations/translations/index_table', translations: current_translations %>
+  </div>
+
+  <% old_translations = [
     {
```
Comment (mirandapedro) [inline WcaOnRails/app/views/regulations/translations/index.html.erb:38]: I believe the Kazakh translation should be between the russian and the japanese ones, right?

Although this was not originally observed when adding Kazakh, I believe we sort first by the most recent translations and then by alphabetical order in english.

**Comment by mirandapedro** [comment]
I believe the Kazakh translation should be between the russian and the japanese ones, right?

Although this was not originally observed when adding Kazakh, I believe we sort first by the most recent translations and then by alphabetical order in english.

**Comment by hanwu85** [comment]
Thanks!
That makes sense, just modified.

File: WcaOnRails/app/views/regulations/translations/index.html.erb [hunk 2]
```diff
@@ -10,35 +10,35 @@
...
-      version: "2020-01-01",
+      version: "2020-08-01",
       language: "简体中文",
       language_english: "Chinese",
       url: "./chinese",
     },
-    {
-      version: "2020-01-01",
+        {
```
Comment (lgarron) [inline WcaOnRails/app/views/regulations/translations/index.html.erb:18]:
```suggestion
    {
```

**Pull request merged**